test/upgrade/git/upgrade_to_nonexistent_pubspec_test.dart

```diff
@@ -1,44 +0,0 @@
-// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
-
-import 'package:test/test.dart';
-
-import '../../descriptor.dart' as d;
-import '../../test_pub.dart';
-
-main() {
-  test("upgrades Git packages to a nonexistent pubspec", () async {
```

Comment:
this test doesn't seem --packages-dir specific?

Reply:
hmm, yeah I am not sure why I thought it was... Restored it